tests/testcontainers/logstash.py

```diff
@@ -23,128 +28,172 @@ class LogstashContainer(DockerContainer):  # type: ignore
         with LogstashContainer() as lc:
             # NOTE: container will terminate once out of this with statement
             url = lc.get_url()
-
-    With custom config
-    ------------------
-    ::
-
-        lgc = LogstashContainer(port=self.logstash_http_port)
```

Comment:
I removed this feature since I opted to always running logstash with ssl, that was the only the case in the previous logstash tests this feature was used.

I don't think we really need to have such level of customisation here, but I'm happy to discuss different opinions :)